lib/compat/wordpress-6.7/rest-api.php

```diff
@@ -60,6 +59,7 @@ function gutenberg_block_editor_preload_paths_6_7( $paths, $context ) {
 		$paths[]          = array( '/wp/v2/global-styles/' . $global_styles_id, 'OPTIONS' );
 
 		// Remove duplicate or unnecessary global styles paths.
+		$excluded_paths   = array();
```

Comment:
Unrelated, but just moving var assignment closer to where it's being used, inside the block.